service/awsconfig/v10/adapter/instances_test.go

```diff
@@ -0,0 +1,143 @@
+package adapter
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v10/cloudconfig"
+)
+
+func Test_Adapter_Instance_RegularFields(t *testing.T) {
```

Comment:
These tests here did not work at all before. There was an issue with the error handling and an error was returned all the time because of the image ID changes which are now bound to the region. The tests I wrote here do actually work now.